.github/workflows/CI.yml

```
@@ -0,0 +1,39 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master

```

```suggestion
      - main
  push:
    branches:
      - main

```

You used the wrong branch name!